app/javascript/spree/dashboard/controllers/webhooks_subscriber_events_controller.js

```diff
@@ -0,0 +1,24 @@
+import { Controller } from "@hotwired/stimulus"
+
+export default class extends Controller {
+  eventsCheckboxes = document.getElementsByName("webhooks_subscriber[subscriptions][]")
+
+  disableCheckboxes() {
+    for (let cb of this.eventsCheckboxes) {
+      cb.disabled = true
+    }
+  }
+
+  enableCheckboxes() {
+    for (let cb of this.eventsCheckboxes) {
+      cb.disabled = false
+    }
+  }
+
+  initialize() {
+    let subscribe_to_all = document.getElementById("subscribe-to-all")
```

Comment:
@aleksandarpetrushev in stimulus you can should use targets for these - https://stimulus.hotwired.dev/reference/targets